tests/infra_test.go

```diff
@@ -366,6 +370,53 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		)
 	})
 
+	Describe("tls configuration", func() {
+
+		It("should be propagated to virt-api, virt-handler and virt-exportpoxy", func() {
+			cipher := tls.CipherSuites()[0]
+			kvConfig := util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
+			kvConfig.TLSConfiguration = &v1.TLSConfiguration{
+				MinTLSVersion: v1.VersionTLS12,
+				Ciphers:       []string{cipher.Name},
+			}
+			tests.UpdateKubeVirtConfigValueAndWait(*kvConfig)
+			newKv := util.GetCurrentKv(virtClient)
+			Expect(newKv.Spec.Configuration.TLSConfiguration.MinTLSVersion).To(BeEquivalentTo(v1.VersionTLS12))
+			Expect(newKv.Spec.Configuration.TLSConfiguration.Ciphers).To(BeEquivalentTo([]string{cipher.Name}))
+
+			tests.EnableFeatureGate("VMExport")
+			testsuite.WaitExportProxyReady()
+			labelSelectorList := []string{"kubevirt.io=virt-api", "kubevirt.io=virt-handler", "kubevirt.io=virt-exportproxy"}
+
+			var podsToTest []k8sv1.Pod
+			for _, labelSelector := range labelSelectorList {
+				podList, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{
+					LabelSelector: labelSelector,
+				})
+				Expect(err).ToNot(HaveOccurred())
+				podsToTest = append(podsToTest, podList.Items...)
+			}
+
+			for i, pod := range podsToTest {
+				func(i int, pod k8sv1.Pod) {
+					stopChan := make(chan struct{})
+					defer close(stopChan)
+					Expect(tests.ForwardPorts(&pod, []string{fmt.Sprintf("844%d:%d", i, 8443)}, stopChan, 10*time.Second)).To(Succeed())
+					tlsConfig := &tls.Config{
```

Comment:
Is it possible to have a negative test case?

Reply:
Sure! done!